### PR TITLE
[small]feature/319

### PR DESCRIPTION
**Pull request by josdem**
- Adding mobile validation format in service and controller
- Setting mobile size as 10

**Screenshot**
Screenshot 2024-09-10 105245

### CODE REVIEW

File: src/main/java/com/josdem/vetlog/command/UserCommand.java [hunk 1]
```diff
@@ -38,10 +38,10 @@ public class UserCommand implements Command {
     @Size(min = 1, max = 100)
     private String lastname;
 
-    @Size(min = 1, max = 250)
-    private String phone;
+    @Size(min = 10, max = 10)
+    private String mobile;
```
Comment (josdem) [inline src/main/java/com/josdem/vetlog/command/UserCommand.java:43]: In the future, we want to have this format forced by UI: (123) 456-7890
From now on, we can go with only numbers.

**Review by kaminuma — CHANGES_REQUESTED** [review]
HI @josdem 
I reviewed the contents of the PR, and it appears that while the CountryCode for user registration is being passed to the backend from the controller, it is not being persisted in the database.

UserBinder class add
`user.setCountryCode(userCommand.getCountryCode());`

**Comment by josdem** [comment]
@all-contributors please add @kaminuma for bug

**Review by kaminuma — APPROVED** [review]
[LGTM]

**Pull request merged**